content/plugins/notes/cost-insights.md

```diff
@@ -0,0 +1,74 @@
+---
+name: cost-insights
+---
+
+## Cost Insights
```

Comment:
Copying content from other websites [can cause SEO problems and should be avoided](https://moz.com/learn/seo/duplicate-content). The content in question is open source, so from a licensing perspective it's fine to do this, but it confuses Google and won't help us to show up on search results.

I would prefer to see a tutorial which shows a user how to use the cost insights plugin, or a short, original, paragraph how to configure it. If you don't feel confident writing something like that, it is better to omit this section entirely.

Please replace this with something original or remove it entirely.